apps/web/src/components/Sidebar.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { Logo } from "./Logo";
+import { cn } from "../utils/classnames";
+import { useQuery } from "react-query";
+import { getPagesQuery } from "../composedb/page";
+import { composeClient } from "../lib/compose";
+import { DecryptedText } from "./DecryptedText";
+import { useCeramic } from "../hooks/useCeramic";
+import { useAccount } from "wagmi";
+
+type SidebarProps = {
+  className?: string;
+};
+
+export function Sidebar({ className }: SidebarProps) {
+  const { isConnected } = useAccount();
+  const ceramic = useCeramic();
+  const myPagesQuery = useQuery(
+    ["PAGES", composeClient.id],
+    async () => {
+      const query = await getPagesQuery();
+      const pages = query.data?.pageIndex?.edges.map((edge) => edge.node) ?? [];
+      const myPages = pages.filter(
+        (page) => page.createdBy.id === composeClient.id
+      );
+      return myPages;
+    },
+    { enabled: isConnected && ceramic.isInitialized }
```

Comment:
Is this enough? What if we don't have a session?

Reply:
Updated to use the same logic as in `create`